.clang-format

```diff
@@ -22,48 +22,3 @@ PointerAlignment: Left
 Standard: c++14
 TabWidth: 2
 UseTab: Never
-IncludeBlocks: Regroup
```

Comment:
Wrong direction for merge chosen here

Reply:
Not really, just reverted the changes short to apply the old clang-format to the conflicted files during merge and forgot to revert changes before commit ;-). Hope it is the last time I have to merge sth into this PR, as rebasing doesn't work (git says every file has a conflict then) and due to other subdirectories I don't touch it is sometimes hard to merge, as I can't just run clang-format without comming before (and suqashing a merge commit doesn't git like).

Should be fixed now

Reply:
Might I recommend another approach? Split the PR into 3 commits: format-changes, format-corrections and non-format changes.

Instead of merging, start a new branch, apply format on it. Then cherry-pick the last 2 commits (or rebase them) to the new branch. Delete old, rename, and ta-da

Reply:
The main problem is the `// clang-format off` changes, as they are part of this initial commit - otherwise I just could rerun clang-format one a clean base - yeah. But I hope this PR will be squashed, so it should be not important if there is a merge or rebase.